tests/MapTest.php

```diff
@@ -1141,6 +1141,36 @@ public function testHasPath()
 	}
 
 
+	public function testIfBlankElse()
+	{
+		Map::from( ['a' => 1, 'b' => 0] )->if(
+			fn(Map $map) => $map->has('c'),
+			function(Map $_) { $this->assertTrue(false); }
+		);
+		$this->assertTrue(true);
+	}
+
+
+	public function testIfElse()
+	{
+		Map::from( ['a' => 1, 'b' => 0] )->if(
+			fn(Map $map) => $map->has('c'),
```

Comment:
Same here

Reply:
resolved in 43ce8d41b8894a4abc84e2be81bd51237264f8ef